src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -168,6 +178,11 @@ public static void Run(ListSourceArgs args, Func<ILogger> getLogger)
                             }
                             legend += " ";
                             getLogger().LogMinimal(legend + source.Source);
+
```

Comment:
Same feedback as above applies. Curious what others think

Reply:
In code I prefer doing Source then Warning, but I think is more readable if we show the warnings at the end

Reply:
> but I think is more readable if we show the warnings at the end

Can we do that then? 

@JonDouglas @chgill-MSFT what do you guys think? 

@martinrrm 

Can you add an example so they can compare the experience.

Reply:
@donnie-msft  @nkolev92 Current behavior, after looking at a longer list I think this is a good solution. I'll try the other way and attach image

![image](https://user-images.githubusercontent.com/43253759/168686895-7eb15fef-d9c8-4d19-a4c8-73ac3e2dcbc2.png)

Reply:
The other way: It looks more clean but not sure if its better @JonDouglas @chgill-MSFT what do you think?
![image](https://user-images.githubusercontent.com/43253759/168687417-f739358a-f3a0-4d80-be84-cb01370eb595.png)

Reply:
I like the 2nd one better, but would love to hear opinions from Jon and Chris.

If you got the direction of 1, we can probably tweak a few things to avoid duplication. 

For example we mention the url, but it's displayed already. It can be a bit redundant.

Reply:
Reviewed these two options with Jon and Chris. We are considering option 2 but we are going to display a list of HTTP sources followed by the warning message instead of one warning message for every HTTP source.

Reply:
Sorry I didn't think of this in my last review, but I do prefer the inline warning (as in https://github.com/NuGet/NuGet.Client/pull/4600/files#r874181365)

... which means I think it's redundant to output the source. The warning could just be:

> 2. nuget2.org [Enabled]
> 
> http://api.nuget.org/v3/i4nd2dex.json
> 
> warn: "Support for 'HTTP' sources will be removed in a future version.